src/main/webapp/scripts/session-script.test.js

```diff
@@ -63,6 +63,28 @@ test('tests copy and paste', () => {
 });
 
 test('addOnClickTo', () => {
+  addOnClickToPageBuilder();
+  const sessionInfoSpan = document.getElementById('session-info-span');
+  const sessionInfoDiv = document.getElementById('session-info-div');
+  const close = document.getElementsByClassName('close').item(0);
+  const sessionIdInput =
+      document.getElementsByClassName('session-id-input').item(0);
+  sessionscript.addOnClickToElements();
+  sessionInfoSpan.click();
+  expect(sessionInfoDiv.style.display).toEqual('block');
+  close.click();
+  expect(sessionInfoDiv.style.display).toEqual('none');
+  document.execCommand = jest.fn();
+  sessionIdInput.click();
+  expect(document.execCommand).toHaveBeenCalledWith('copy');
+});
+
+/**
+ * Builds a mini-webpage to be used to test addOnClickToElements.
+ * Adds elements with specific ids/class names that the session-script 
+ * function adds onClick listeners to.
+ */
+function addOnClickToPageBuilder() {
```

Comment:
This function name doesnt really match the comment above it. Try to make your functions have action names. Consider: buildTestPage(). It just builds a test page, right? The fact that its for a particular test case doesnt matter, yeah?

Reply:
the content itself is pretty specific to just that test

Reply:
Thats fine. buildTestPage() is fine